var/spack/repos/builtin/packages/ace/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from os import environ as env
+from spack import *
+
+
+class Ace(Package):
+    """ACE is an open-source framework that provides many components and
+       patterns for developing high-performance, distributed real-time and
+       embedded systems. ACE provides powerful, yet efficient abstractions
+       for sockets, demultiplexing loops, threads, synchronization
+       primitives."""
+
+    homepage = "http://www.dre.vanderbilt.edu/~schmidt/ACE.html"
+    url = "http://download.dre.vanderbilt.edu/previous_versions/" \
+          "ACE-6.5.1.tar.gz"
```

Comment:
You can put all of this on one line, Spack exempts URLs from the line length checks.